### ISSUE TITLE
Tags? Is this jsr181/jsr181-api or javax.jws

### ISSUE THREAD

**Issue by wltjr** [javaee/javax.jws#1]
I need this as a dependency of [spring-web](https://github.com/spring-projects/spring-framework/blob/master/spring-web/spring-web.gradle#L32) I cannot tell if this is [jsr181/jsr181-api](http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22javax.jws%22) or [javax.jws](http://search.maven.org/#search%7Cga%7C1%7Ca%3A%22javax.jws%22).

I see a 1.1 snapshot, so that would match versioning of jsr181. I do not see the [javax.jws-api artifact id per the pom.xml](https://github.com/javaee/javax.jws/blob/master/pom.xml#L53) in [maven central repo](http://search.maven.org/#search%7Cga%7C1%7Cjavax.jws-api).

I can grab a snapshot tag for my needs, but just curious in general if this is the proper home. If tags can be added, etc. thanks!

**Comment by Stexxen** [comment]
Yep, same problem. I'm using JDK 11 and this javax.jws-api has a module-info but is not in the Maven Repos.

**Comment by Stexxen** [comment]
It's not yet arrived in the /releases/ but is available in /staging/

```
    <repositories>
        <repository>
            <id>jvnet-nexus-staging</id>
            <url>http://maven.java.net/content/repositories/staging/</url>
            <layout>default</layout>
        </repository>
    </repositories>

    <dependencies>
        <dependency>
            <groupId>javax.jws</groupId>
            <artifactId>javax.jws-api</artifactId>
            <version>1.1</version>
        </dependency>
    </dependencies>
```

**Comment by wltjr** [comment]
@Tomas-Kraus I think we can close this, you created a 1.1 tag on Jun 25. I see you opened the issue with the new repo location.
